### PR TITLE
fix(app-store): Open app-store links in a new tab.

### LINKED ISSUE #5079: App/play store links on SMS screen should open in new tabs
Currently the app/play store badges on SMS screen do not open in new tabs, but should.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #5079

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - r?

File: app/scripts/templates/marketing_snippet.mustache [hunk 1]
```diff
@@ -16,7 +16,7 @@
           <br>
           {{#t}}Download Firefox for Android.{{/t}}
         </p>
-        <a class="marketing-link marketing-link-android" href="{{ downloadLinkAndroid }}" data-marketing-id="{{ marketingId }}" data-marketing-type="android"><img width="152" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
+        <a target="_blank" class="marketing-link marketing-link-android" href="{{ downloadLinkAndroid }}" data-marketing-id="{{ marketingId }}" data-marketing-type="android"><img width="152" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
```
Comment (philbooth) [inline app/scripts/templates/marketing_snippet.mustache]: Tangential question: the `alt` text for iOS finishes with an exclamation mark but the `alt` text for Android finishes with a period. Is that deliberate? Seems funny.

**Comment by philbooth** [comment]
r+

File: app/scripts/templates/marketing_snippet.mustache (hunk 1 above)
Comment (shane-tomlinson) [inline app/scripts/templates/marketing_snippet.mustache]: > Tangential question: the alt text for iOS finishes with an exclamation mark but the alt text for Android finishes with a period. Is that deliberate? Seems funny.

@ryanfeeley - expected or should we make these consistent?

**Pull request merged**